performance-timeline/po-disconnect.html

```diff
@@ -30,15 +30,16 @@
 
   async_test(function (t) {
     var observer = new PerformanceObserver(
-        t.step_func(function(entryList, obs) {
-          checkEntries(entryList.getEntries(),
-            [{ entryType: "mark", name: "mark1"}]);
-          t.done();
+        t.step_func(function (entryList, obs) {
+          assert_unreached("This callback must not be invoked");
```

Comment:
Agreed that the test process is a separate thing from current issue.
I'll let Ilya / Todd chime in here on whether we should update the test (and break test for chrome as a side effect) or wait until spec update for takeRecords etc.